examples/conf/druid/single-server/medium/overlord/runtime.properties

```diff
@@ -0,0 +1,26 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+druid.service=druid/overlord
```

Comment:
Out of curiosity, why not run Coordinator asOverlord and move towards the world where the coordinator and overlord are one

Reply:
I initially felt like the simplicity benefit wasn't that high in this case (since premade configs and startup scripts are being provided), and felt that it would be closer to a full clustered deployment for eventual migration with them separate.

I updated the PR to use that setting, since it would be fine to set that in a larger cluster as well.